tools/tune/dcblock/dcblock_get_abi.m

```diff
@@ -2,7 +2,8 @@
 
 %% Use sof-ctl to write ABI header into a file
 abifn = 'dcblock_get_abi.bin';
-cmd = sprintf('sof-ctl -g %d -b -o %s', setsize, abifn);
+cmd = sprintf('./sof-ctl -g %d -b -o %s', setsize, abifn);
+cd ./../../build_tools/ctl/
```

Comment:
i think we should call the full path rather than change the directory and leave the user in a random location if we fail

Reply:
@cujomalainey, what do you mean by full path instead of change in directory ?

Present the directory for script example_drc.m execution is  ./../tools/tune/drc  before executing the  system(cmd)
system(cmd) is expected to execute ./. ./build_tools/ctl/   directory 

We are hitting the failure  in below lines 

fh = fopen(abifn, 'r'); as fh is -1
if fh < 0
	error("Failed to get ABI header. Is sof-ctl installed?");
end

change in the directory is needed to execute the system(cmd) to generate the *_get_abi.bin. which is located in the build_tools/ctl/ otherwise we get a failure for fh <0 which itself is an error and the current files are not executing the scripts in the present state.

Reply:
Yes, use full path in system command rather than changing directory. I think easily one of the most frustrating things when debugging is when your script leaves you in another directory without your knowledge.